CMSgov/mint-app
# [EASI-3129] FE  -- threaded replies

# EASI-3129

## Changes and Description

- Change from Question verbiage to Discussion verbiage per Figma
- Replace accordions of "Answered/Unanswered" to "New discussions/discussions"
- Similar rendering for multiple replies as a single reply
- Hide/View more replies
- Replying to a thread should show entire thread when answering
- This includes the hide/view more replies
- After Replying to thread the reply get populated on same page
- The user can then choose to go back to main discussion panel
- Unit tests
- Cypress tests



## How to test this change

1. Navigate to discussion panel
2. Ask a question
3. Reply to question
4. Ensure verbiage and flow matches figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
